app/javascript/src/components/product-card.vue

```diff
@@ -42,6 +42,29 @@
       <p class="mt-5 text-justify text-md">
         {{ product.description }}
       </p>
+      <div class="absolute bottom-0 mb-5">
+        <p class="pb-2 text-xs font-bold uppercase">
+          Escoge tu opción
+        </p>
+        <div
+          class="inline"
+          v-for="(categoryProduct, index) in category.products"
+          :key="index"
+        >
+          <button
+            class="w-8 h-8 mx-1 text-gray-600 border border-gray-400 border-solid rounded-full shadow"
+            :class="{'border-primary' : categoryProduct === product }"
+            @click="$emit('change-slide', index)"
+          >
+            <span
+              class="inline text-xs font-bold text-center"
+              :class="{'text-black' : categoryProduct === product }"
+            >
+              {{ "$".repeat(index+1) }}
+            </span>
```

Comment:
No pasá acá que hayan muchos productos y el botón quede $$$$$$$$?
también le faltan los espacios jeje
```suggestion
            >
              {{ "$".repeat(index + 1) }}
            </span>
```

Reply:
sí, de hecho a futuro los '$' se distribuirán por rangos de precio más que por cantidad de items, pero está previsto :)
Lo otro lo corrijo altiro, gracias!